common/api-review/app-exp.api.md

```diff
@@ -4,12 +4,30 @@
 
 ```ts
 
-import { FirebaseApp } from '@firebase/app-types';
-import { FirebaseAppConfig } from '@firebase/app-types';
-import { FirebaseOptions } from '@firebase/app-types';
+import { Component } from '@firebase/component';
+import { FirebaseApp } from '@firebase/app-types-exp';
+import { FirebaseAppConfig } from '@firebase/app-types-exp';
+import { FirebaseOptions } from '@firebase/app-types-exp';
 import { LogCallback } from '@firebase/logger';
 import { LogLevel } from '@firebase/logger';
 import { LogOptions } from '@firebase/logger';
+import { Name } from '@firebase/component';
+import { Provider } from '@firebase/component';
+
+// @internal (undocumented)

```

Comment:
Looked at the docs for `api-extractor`/`api-documenter` and still not sure, how does this show up in the docs? Do either the `@internal` tag or the `(undocumented)` status prevent this from showing up in the docs? It seemed like `@internal` methods still show up in docs but they just say `@internal`? I assume we want it to not be in the docs at all?